### PR TITLE
Changes vars to let/const

### PR DESCRIPTION
**Pull request by frankcash**
Changes `var` to `let` or `const`

### CODE REVIEW

File: tools/buildPackage.js [hunk 1]
```diff
@@ -2,21 +2,21 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-var execute = require('./execute')
+let execute = require('./execute')
```
Comment (diracdeltas) [inline tools/buildPackage.js]: this can be const

Comment (frankcash) [reply tools/buildPackage.js]: Fixed

File: tools/buildPackage.js [hunk 2]
```diff
@@ -2,21 +2,21 @@
...
 const isWindows = process.platform === 'win32'
 const isDarwin = process.platform === 'darwin'
-var arch = 'x64'
+let arch = 'x64'
 const isLinux = process.platform === 'linux'
 
-var pack = JSON.parse(fs.readFileSync('package.json', 'utf-8'))
-var electronPrebuiltPack = JSON.parse(fs.readFileSync('./node_modules/electron-prebuilt/package.json', 'utf-8'))
+let pack = JSON.parse(fs.readFileSync('package.json', 'utf-8'))
+let electronPrebuiltPack = JSON.parse(fs.readFileSync('./node_modules/electron-prebuilt/package.json', 'utf-8'))
```
Comment (diracdeltas) [inline tools/buildPackage.js]: these can also be const

File: tools/buildPackage.js [hunk 3]
```diff
@@ -30,11 +30,11 @@ if (isWindows) {
 
 const buildDir = 'Signal-' + process.platform + '-' + arch
 
-var env = {
+let env = {
```
Comment (diracdeltas) [inline tools/buildPackage.js]: const

File: tools/execute.js [hunk 4]
```diff
@@ -1,23 +1,23 @@
-var exec = require('child_process').exec
+let exec = require('child_process').exec
```
Comment (diracdeltas) [inline tools/execute.js]: const

File: tools/execute.js [hunk 5]
```diff
@@ -1,23 +1,23 @@
...
 
-  for (var key in env) {
+  for (let key in env) {
     if (env.hasOwnProperty(key)) {
       process.env[key] = env[key]
     }
   }
 
-  var r = exec(cmd, {
+  let r = exec(cmd, {
```
Comment (diracdeltas) [inline tools/execute.js]: const

**Pull request merged**